my_game.py

```diff
@@ -134,7 +128,7 @@ def reset(self):
     def is_invincible(self):
         return self.invincibility_timer > 0
 
-    def on_update(self, delta_time: float = 1 / 60):
+    def update(self, delta_time: float = 1 / 60):
```

Comment:
1: _on_update_ is the standard name for an update function which takes _delta_time_.
2: Having a default for _delta_time_ is weird. Why would one want that?

Reply:
yes, i think that having an on_update function for a sprite is bad practice in arcade...
So we might have to move the invincibility check to ingamewiev?

Reply:
right, it is not bad practise - thought sprite didn't have on_update.
However the function apparently needs us to fill in delta_time, so calling the function will look like this:
`# update player sprite
self.player_sprite.on_update(delta_time)`

Reply:
Yes, and that is what we want (We really need to do it everywhere). If we don't use delta time, the "speed" of the game will vary between computers.

Reply:
Please remove the _1/60_ default value for _delta_time_:
```
def on_update(self, delta_time)
```